downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -164,7 +164,36 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
         }
     }
     ```
+
+- A bug where the runtime error messages were including the member types of a union type instead of the original union type name has been fixed.
+  
+  ```ballerina
+    type AC map<int>|string;
+    
+    public function main() returns error? {
+        map<anydata> x = {a: "Ballerina"};
+        AC|boolean _ = check x.cloneWithType();
+    }
+  ```
+
+  The above program now fail with the following runtime error
   
+  ```
+  error: {ballerina/lang.value}ConversionError {"message":"'map<anydata>' value cannot be converted to '(AC|boolean)': 
+                  {
+                    map field 'a' should be of type 'int', found '"Ballerina"'
+                  }"}
+  ```
+  
+  Instead of the following. Note the original union type name `AC` is missing in the error message.
+  
+  ```
+  error: "{ballerina/lang.value}ConversionError",message="'map<anydata>' value cannot be converted to '(map<int>|string|boolean)': 
+                  {
+                    map field 'a' should be of type 'int', found '"Ballerina"'
+                  }"
+  ```
```

Comment:
I would keep this simple and say 

```
This now fails with "'map<anydata>' value cannot be converted to '(AC|boolean)'" instead of "'map<anydata>' value cannot be converted to '(map<int>|string|boolean)'".
```